box.py

```diff
@@ -270,7 +270,7 @@ def __dir__(self):
         items = set(dir(dict) + ['to_dict', 'to_json',
                                  'from_json', 'box_it_up'])
         # Only show items accessible by dot notation
-        for key in self.keys():
+        for key in list(self.keys()):
```

Comment:
Same case with `.keys` as `.items`, it's an iterable in Python 3, but not in python 2, so for safety 2to3 casts them to lists which is unnecessary and much slower.